index.html

```diff
@@ -5,10 +5,21 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link rel="stylesheet" href="styles.css">
+    @import url('https://fonts.googleapis.com/css2?family=Bungee+Inline&display=swap');
     <title>http-app-project</title>
 </head>
 <body>
+<section class="form-container">
```

Comment:
great structuring in form layout, the class name defines what the container will keep inside.

good start on button for soon to be displayed giphy ✨